bundles/org.openhab.ui/web/src/components/app.vue

```diff
@@ -272,12 +272,13 @@ import DeveloperSidebar from './developer/developer-sidebar.vue'
 
 import auth from './auth-mixin.js'
 import i18n from './i18n-mixin.js'
+import sseEvents from './sse-events-mixin.js'
```

Comment:
Let's drop the .js extension here, as for most JS imports in MainUI:

```suggestion
import sseEvents from './sse-events-mixin'
```